django/db/migrations/executor.py

```diff
@@ -186,6 +194,16 @@ def detect_soft_applied(self, project_state, migration):
                 if model._meta.db_table not in self.connection.introspection.table_names(self.connection.cursor()):
                     return False, project_state
                 found_create_migration = True
-        # If we get this far and we found at least one CreateModel migration,
+
+            if isinstance(operation, migrations.AddField):
+                table = "%s_%s" % (migration.app_label, operation.model_name)
```

Comment:
You need to get the table out of the model this field belongs to. See the `isinstance(operation, migrations.CreateModel)` branch above.